### PR TITLE
[LincombOperator] implement 'apply_inverse' 

### PR DESCRIPTION
**Pull request by sdrave**
and 'apply_inverse_transpose' for the special case of len(self.operators) == 1.

See discussion in #272.

### CODE REVIEW

File: src/pymor/operators/constructions.py [hunk 1]
```diff
@@ -147,6 +147,34 @@ def jacobian(self, U, mu=None):
...
 
+    def apply_inverse(self, V, mu=None, least_squares=False):
+        if len(self.operators) == 1:
+            if self.coefficients[0] == 0.:
+                if least_squares:
+                    return self.source.zeros(len(V))
+                else:
+                    raise InversionError
+            else:
+                U = self.operator.apply_inverse(V, mu=mu, least_squares=least_squares)
```
Comment (pmli) [inline src/pymor/operators/constructions.py]: `self.operator` -> `self.operators[0]`

File: src/pymor/operators/constructions.py [hunk 2]
```diff
@@ -147,6 +147,34 @@ def jacobian(self, U, mu=None):
...
+
+    def apply_inverse_transpose(self, U, mu=None, least_squares=False):
+        if len(self.operators) == 1:
+            if self.coefficients[0] == 0.:
+                if least_squares:
+                    return self.range.zeros(len(U))
+                else:
+                    raise InversionError
+            else:
+                V = self.operator.apply_inverse_transpose(U, mu=mu, least_squares=least_squares)
```
Comment (pmli) [inline src/pymor/operators/constructions.py]: See above.

**Comment by sdrave** [comment]
Oh dear, we really need a faster test suite ... - Should be fixed now.

**Comment by pmli** [comment]
When merging with master, everything works now.

**Pull request merged**